src/domain/accounts/index.types.d.ts

```diff
@@ -136,33 +137,13 @@ type AccountValidator = {
   validateWalletForAccount(wallet: Wallet): true | ValidationError
 }
 
-type NewAccountWithEmailIdentifier = {
-  kratosUserId: KratosUserId
-  phone?: undefined
-  phoneMetadata?: undefined
-}
-
-type NewAccountWithPhoneIdentifier = {
-  kratosUserId: KratosUserId
-  phone: PhoneNumber
-  phoneMetadata?: PhoneMetadata
-}
-
-type NewAccount = NewAccountWithEmailIdentifier | NewAccountWithPhoneIdentifier
-
 interface IAccountsRepository {
+  list(): Promise<Account[] | RepositoryError>
```

Comment:
This seems like a potential performance issue. RU sure we want to expose being able to list _all_ accounts? Without a filter criteria or pagination (ie. iterator pattern)?

Reply:
I don't think this is actually been used, only the function below `listUnlockedAccounts` is being used based on a quick search. I have removed `list` 

`listUnlockedAccounts` should indeed probably get some pagination but it's currently only used once a day for cronjob so I think this can wait until it's needed